### PR TITLE
#61: Create restaurant details page

### LINKED ISSUE #61: Create 'Individual Restaurant Details' page
The Restaurant Details page provides the user more information about the individual restaurant shown in their recommendations such as name, description, location/directions, and online reviews. 

This task is a part of User story #23.

insights

### PR DESCRIPTION
**Pull request by thomastai1666**
Adds the restaurant details page which closes #61. Uses google maps API to show the directions to the restauraunt.

Uses Documenu data schema defined here: https://documenu.com/docs#get_search_restaurants

Screen Shot 2021-11-01 at 9 03 46 PM

### CODE REVIEW

File: front-end/src/pages/RestaurantDetails.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.RestaurantDetails {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
```
Comment (Jen-Lopez) [inline front-end/src/pages/RestaurantDetails.css:5]: When you view the page on mobile (when you navigate from home -> restaurant details), it seems like the page overflows. Changing `width` to `min-width` seems to solve the issue though. 

**BEFORE**
before

**AFTER**
after

**Review by Jen-Lopez — CHANGES_REQUESTED** [review]
Just a minor fix!

File: front-end/src/pages/RestaurantDetails.css (hunk 1 above)
Comment (thomastai1666) [reply front-end/src/pages/RestaurantDetails.css:5]: Thank you, that looks so much better! 
```suggestion
  min-width: 100%;
```

**Review by Jen-Lopez — APPROVED** [review]

**Review by Golemwardox — APPROVED** [review]
LGTM.

**Pull request merged**